tsconfig.test.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "declaration": true,
```

Comment:
You don't need to produce declaration files or anything extra, you're just throwing away the result of the compilation anyway.